solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/util/FlowReceiverContainer.java

```diff
@@ -97,6 +103,9 @@ public UUID bind() throws JCSMPException {
 				return newFlowReceiverReference.getId();
 			}
 		} finally {
+			if (xmlMessageMapper != null) {
+				xmlMessageMapper.resetIgnoredProperties(id.toString());
+			}
```

Comment:
this should cleared only when we bind a new flow. i.e. Right before the `bindCondition.signalAll()` from the `if (existingFlowReceiverReference == null)` path.